.typos.toml

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Zerocracy
```

Comment:
@Yegorov it's better to put this file into the `.github/` directory

Reply:
@yegor256 ok, thanks, changed.